include/wx/menuitem.h

```diff
@@ -122,6 +122,13 @@ class WXDLLIMPEXP_CORE wxMenuItemBase : public wxObject
     // set the accel for this item - this may also be done indirectly with
     // SetText()
     virtual void SetAccel(wxAcceleratorEntry *accel);
+
+    // add the accel to extra accels list
+    void AddExtraAccel(wxAcceleratorEntry *accel);
```

Comment:
What's the reason for taking a pointer here? I.e. why not
```suggestion
    void AddExtraAccel(const wxAcceleratorEntry& accel);
```
?